src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html

```diff
@@ -182,14 +183,19 @@ <h2>Edit seat</h2>
           <div hidden$="[[_checkSeatTaken(seatToEdit)]]">
             <h4>Seat currently taken by:</h4>
             <p>[[seatToEdit.ticket.owner.profile.displayName]]</p>
-            
           </div>
           <div hidden$="[[!_checkSeatTaken(seatToEdit)]]">
             <p>Seat is still available</p>
+            <p>Want to assign a user to a seat?</p>
+            <p>input the ticket ID below</p>
+            <paper-input label="ticket id" value="{{assignId}}"></paper-input>
```

Comment:
Can be a one-way binding

```suggestion
            <paper-input label="ticket id" value="[[assignId]]"></paper-input>
```